libs/components/src/next/title/vl-title.component.ts

```diff
@@ -0,0 +1,65 @@
+import { html, LitElement } from 'lit-element';
+import { choose } from 'lit/directives/choose.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { CSSResult, PropertyDeclarations } from 'lit';
+import { resetStyle } from '@domg/govflanders-style/common';
+import titleStyle from './vl-title.css';
+import { webComponent } from '@domg-wc/common-utilities';
+import { globalStylesNext } from '@domg-wc/common-utilities/css/global-styles-decorator';
+
+export const titleDefaults = {
+    type: 'h1' as string,
+    underline: false as boolean,
+    alt: false as boolean,
+    noSpaceBottom: false as boolean,
+} as const;
+
+@globalStylesNext()
+@webComponent('vl-title-next')
+export class VlTitleComponent extends LitElement {
+    private type: string | null = null;
+    private underline = false;
+    private noSpaceBottom = false;
+    private alt = false;
+
+    static get styles(): (CSSResult | CSSResult[])[] {
+        return [resetStyle, titleStyle];
```

Comment:
resetStyle is niet nodig, ik zou de specifieke reset styles in de titleStyle steken, zie vl-link als voorbeeld.

Reply:
Specifieke reset styles zitten in `titleStyle`, dit moest hier idd niet nog eens instaan.